docs/templates/eia860_child.rst.jinja

```diff
@@ -9,29 +10,40 @@ and planned entities with one or more megawatt of capacity. The form also contai
 information regarding environmental control equipment and construction cost data from
 2013-2018.
 
-As of 2019, the EIA-860 Form is organized into the following schedules:
+As of 2023, the EIA-860 Form is organized into the following schedules:
 
-    * **Schedule 1:** Identification
-    * **Schedule 2:** Power plant data
-    * **Schedule 3:** Generator information
-    * **Schedule 4:** Ownership of generators
-    * **Schedule 6:** Boiler information
+* **Schedule 1:** Identification
+* **Schedule 2:** Power plant data
+* **Schedule 3:** Generator information
+* **Schedule 4:** Ownership of generators
+* **Schedule 6:** Information on Boilers and Associated Equipment
 
 (Schedule 5 contained generator construction cost information)
 {% endblock %}
 
 {% block downloadable_pdfs %}
+
 {% for filename in download_paths %}
-    * :download:`{{ filename.stem.replace("_", " ").title() }} (PDF) <{{ filename }}>`
+* :download:`{{ filename.stem.replace("_", " ").title() }} (PDF) <{{ filename }}>`
+
 {% endfor %}
 {% endblock %}
 
 {% block accessible %}
-EIA-860 data stretches back to 2001, and PUDL currently covers all years starting from 2004. The prior years are published as DBF files and need a special process to read and
-extract. We intend to include these older years as soon as we can.
+EIA-860 data goes back to 2001, and PUDL incorporates most of the data from all those
+years. From 2001-2003 the data was distributed as VisualFoxPro DBF files. Starting in
+2004 the data is distributed as Microsoft Excel spreadsheets.
+
+Earlier data from the EIA-860A (utility) and EIA-860B (non-utility) forms exists for the
+years 1990-2000, but is not yet included in PUDL.
+
+PUDL does not yet include the EIA-860 data reporting:
+
+* Wind and Solar generators (Schedules 3.2 and 3.3)
+* Energy storage (Schedule 3.4)
+* Multi-fuel generators (Schedule 3.5)
+* Environmental Control Equipment (Schedule 6) has been partially incorporated.
```

Comment:
We just haven't gotten around to it, or had anyone knocking down the doors to get them. But it would be good to pull them in, and doing so is much easier these days!